### DISCUSSION TITLE
Does it support another browser ?

### DISCUSSION THREAD

**Discussion by ravindra-LL** [Skyvern-AI/skyvern#1167] [Q&A]
is this tool also run the tests on another browser like chrome ??

**Comment by suchintan** [comment]
Yep! I believe you can have Skyvern run on Chrome (instead of Chromium) with a 1-line change here: https://github.com/Skyvern-AI/skyvern/blob/20f30d12245dd94cffe850e8dae452d2569a7270/skyvern/webeye/browser_factory.py#L266
